lib/followers/getUsersFollowing.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
Use promise resolve to avoid disabling this (see https://eslint.org/docs/latest/rules/no-await-in-loop)